wpilibc/athena/include/Joystick.h

```diff
@@ -8,9 +8,10 @@
 #ifndef JOYSTICK_H_
 #define JOYSTICK_H_
 
-#include <cstdint>
+#include <stdint.h>
```

Comment:
It's a C system header, so it goes before C++ system headers. There should be a newline separating it from the C++ ones though.